tasks/parse-build-deploy-script/parse-build-deploy-script.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: parse-build-deploy-script
+spec:
+  description: >-
+    Tekton task to get the env variables of a workspace using build_deploy.sh
+  workspaces:
+    - name: source
+  params:
+    - default: "source"
+      description: directory in the `output` workspace to clone the repo into.
+      name: subdirectory
+      type: string
+    - name: path-context
+      default: tests/insights-dashboard-master
+      description: path context directory inside the repo
+      type: string
+  results:
```

Comment:
Add the node-build-version variable as well

Reply:
is it a param or should it be retrieved from a script? (it's not in build_deploy.sh)